src/test/java/de/tum/in/www1/artemis/VideoUnitIntegrationTest.java

```diff
@@ -72,13 +72,15 @@ public void testAll_asStudent() throws Exception {
     @Test
     @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
     public void createVideoUnit_asInstructor_shouldCreateVideoUnit() throws Exception {
+        videoUnit.setSource("https://www.youtube.com/embed/8iU8LPEa4o0");
```

Comment:
A bit disappointed that you didn't select https://www.youtube.com/embed/dQw4w9WgXcQ ;)